src/typeahead/typeahead-window.ts

```diff
@@ -4,8 +4,9 @@ import {NgbHighlight} from './highlight';
 @Component({
   selector: 'ngb-typeahead-window',
   exportAs: 'ngbTypeaheadWindow',
+  host: {'class': 'dropdown-menu', 'style': 'display: block'},
   template: `
-    <div class="dropdown-menu" aria-labelledby="dropdownMenu1">
+    <div aria-labelledby="dropdownMenu1">
```

Comment:
Not sure I understand this change. If we are moving things to the `host` (which is a good move IMO) let's move everything. What is a reason for this change?

BTW: here `dropdownMenu1` is hard-coded we should pass it as input.

I can do those fixes in a separate PR but I would like to know what is the reasoning behind moving this to `host`?